cli/src/commands/list.ts

```diff
@@ -24,35 +23,55 @@ export default class List extends Command {
     }),
   };
 
+  protected getListTable(isLocked: boolean) {
+    const listTable = {
+      columns: ['name', 'updatedAt', 'URL'],
+      config: {
+        name: {
+          headingTransform: () => 'APPLICATION',
+          minWidth: 25,
+        },
+        updatedAt: {
+          headingTransform: () => 'LAST UPDATED',
+          minWidth: 25,
+        },
+      },
+    };
+    if ( isLocked ) {
+      listTable.columns.push('lockReason');
+      const lockTmp = {
+        lockReason: {
+        headingTransform: () => 'LOCK REASON',
+        minWidth: 25,
+      },
+    };
+      return Object.assign(listTable, lockTmp);
+    } else {
+      return listTable;
```

Comment:
We try to code with an "early return" strategy, or with a single return.  Having
```ts
if (...) {
  // many lines
} else {
  return x;
}
```
is hard to read when there are "many" lines, because the indentation suddenly matters a lot and I need to correlate "`return x`" with the condition in the "`if`" many lines above.

In this case I would suggest using e.g.
```ts
if (isLocked) {
  listTable.columns.push('lockReason');
  listTable.lockReason = { ... };
}
return listTable;
```
which avoids some indentation issues.